web/styles/popovers.css

```diff
@@ -317,6 +317,13 @@ ul.popover-group-menu-member-list {
     }
 }
 
+ul.popover-group-menu-member-list:not(:has(li))::after {
+    content: "No members in the group" !important;
+    color: var(--color-text-search-placeholder);
+    padding-left: 10px;
+    font-style: italic;
+}
+
```

Comment:
This isn't the correct approach to solving this issue. It is way too hacky with the CSS definition. Please see #30978 for a better approach to the issue.

Please also go through our [contributing guideline](https://zulip.readthedocs.io/en/stable/contributing/index.html) to learn how to contribute effectively.